examples/nlp/pretraining_BERT.py

```diff
@@ -0,0 +1,481 @@
+"""
+Title: Pre-training BERT with 🤗 Transformers.
```

Comment:
Use "Hugging Face", do not include emojis in titles or descriptions.

Reply:
Changed this!